docs/dev-standards/index.md

```diff
@@ -0,0 +1,32 @@
+# Development Standards
+
+## Introduction
+
+At UKHSA, we build and operate digital services that support public health outcomes. To ensure these services are secure, reliable and maintainable, we follow a consistent set of engineering standards. These standards reflect industry best practice and are tailored to the unique needs of public sector delivery, including regulatory compliance, data protection and operational resilience.
+
+This guidance provides a shared foundation for how we design, build, test and operate software. It promotes collaboration across disciplines, supports continuous improvement and ensures that quality and security are embedded from the outset — not added as an afterthought.
```

Comment:
Small point, but rather than using a literal em-dash character, we're using --- that gets converted to an em dash during HTML conversion.
```suggestion
This guidance provides a shared foundation for how we design, build, test and operate software. It promotes collaboration across disciplines, supports continuous improvement and ensures that quality and security are embedded from the outset --- not added as an afterthought.
```